.github/workflows/i18n-unused-keys.yml

```
@@ -0,0 +1,84 @@
+  pull_request:
+    paths:
+      - "client/src/**"
+
+jobs:
+  detect-unused-i18n-keys:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write  # Required for posting PR comments
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)